standalone/docker-entrypoint.sh

```diff
@@ -70,6 +70,17 @@ for key in ${!jmenv[@]}; do
     sed -i "s/^#$key =.*/$key = $val/g" "$CONFIG" || echo "Couldn't set : $key = $val, please modify $CONFIG manually"
 done
 
+BASIC_AUTH_USER=${APP_USER:-joinmarket}
+BASIC_AUTH_PASS=${APP_PASSWORD:-}
+
+if [ "${BASIC_AUTH_PASS}" = "" ]; then
+    BASIC_AUTH_PASS=$(openssl rand -hex 32)
+    echo "Basic auth credentials: user=${BASIC_AUTH_USER} password=${BASIC_AUTH_PASS}"
+fi
```

Comment:
Not happy with printing the password to log files.. should it just fail if no password is provided? What do you think?

Reply:
Hm yeah so the standalone setup is used in umbrel only at the moment right? So we could make the password required? Or make it so that there's either a password and the access will be protected or there is none and the access will be unprotected?

Reply:
Kind of, but it also should be used for Citadel and possibly the dev regtest setup.
Making the password required would be the simpler option.
If we do not want to enable Basic Auth at all, we'd need a "more complicated" approach, e.g. two nginx configs and copy either the one or the other - but this seems avoidable for now. Hope to keep it as simple as possible.

Keep in mind that if no password is provided a different one will be generated on every startup.

Reply:
Ok I'd say for the sake of simplicity let's make the password required for now. Longer term we'll move a way from basic auth anyway imo and move towards a real in-app password screen like Thunderhub has. When we do that we can think about required-non-required passwords again.